### DISCUSSION TITLE
Related to postman collection generating from yml file

### DISCUSSION THREAD

**Discussion by shubhambajad** [apideck-libraries/portman#329] [Q&A]
Hi Team,

I have schema.yml file which consist of all the schema related to APIs.
For post api it was giving the schema body in three format for which I have mentioned below.

requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Group'
          application/x-www-form-urlencoded:
            schema:
              $ref: '#/components/schemas/Group'
          multipart/form-data:
            schema:
              $ref: '#/components/schemas/Group'
        required: true

When I am converting this yml file to postman collection using a portman tool it is only considering a application/x-www-form-urlencoded format not the other format and i want to convert the payload in application/json format so is there any way to do this?

**Comment by shubhambajad** [comment]
@thim81 do you know anything about it ?

**Reply by thim81** [reply]
@shubhambajad In the upcoming version, we have improved the "content-type" handling of Portman, so potentially this strange behaviour would be solved.

**Reply by shubhambajad** [reply]
@thim81 One more question like, currently we have provided a overwrite feature for application/json payload so does portman new feature will provide support for overwriting a other content-type as well like "application/x-www-form-urlencoded"? 

Also, till what time new version of Portman will get deploy?

**Reply by thim81** [reply]
The main supported content-type is still `application/json`. The feature request for [application/x-www-form-urlencoded](https://github.com/apideck-libraries/portman/issues/325) is still open.

The new version of Portman is ready to go and should be shipped this week.
